modules/340-monitoring-yandex-cloud/templates/yandex-cloud-metrics-exporter/nat-instance-service-monitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.monitoringYandexCloud.internal.natInstanceName (not .Values.monitoringYandexCloud.disableScrapeNATInstanceMetrics) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Consider switching to pod monitor without service

Reply:
done